packages/datagrid/stories/datagrid.component.js

```diff
@@ -175,4 +177,41 @@ storiesOf('Component Datagrid', module)
 		}
 		return <WithLayout />;
 	})
-	.add('dynamic change data', () => <DynamicDataGrid />);
+	.add('@deprecated - dynamic change data with forceRedrawRows', () => (
+		<DynamicDataGrid forceRedraw />
+	))
+	.add('dynamic change data', () => <DynamicDataGrid />)
+	.add('faster datagrid', () => (
+		<div style={{ height: '100vh' }}>
+			<IconsProvider />
+			<p>
```

Comment:
All this description is an explanation of the implementation. Remove that from the story please, and add it to the PR description please